specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/livySpark.json

```diff
@@ -725,22 +728,27 @@
       "type": "object",
       "properties": {
         "id": {
+          "description": "The session id.",
           "format": "int32",
           "type": "integer"
         },
         "appId": {
+          "description": "The application id of this session.",
           "type": "string"
         },
         "appInfo": {
+          "description": "The detailed application info.",
           "type": "object",
           "additionalProperties": {
             "type": "string"
           }
         },
         "state": {
-          "type": "string"
+          "description": "The batch state.",
+          "$ref": "#/definitions/SessionState"
```

Comment:
>SessionState [](start = 33, length = 12)

I noticed that spark batch job and spark session job share the same SessionState. Could you please double check this?
If they don't have the totally same state please split it.